exasol_script_languages_developer_sandbox/lib/logging.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from enum import Enum
+
+from rich.logging import RichHandler
+
+SUPPORTED_LOG_LEVELS = {"normal": logging.WARNING, "info": logging.INFO, "debug": logging.DEBUG}
+
+
+class LogType(Enum):
+    SETUP = "setup"
+    EXPORT = "export"
+    VM_BUCKET = "vm_bucket"
+    CI_CODEBUILD = "ci_codebuild"
+    AWS_ACCESS = "aws_access"
+    ANSIBLE = "ansible"
+
+
+def get_status_logger(log_type: LogType) -> logging.Logger:
+    return logging.getLogger(f"eslds-{log_type.value}")
```

Comment:
What is eslds

Reply:
**E**xasol-**S**cript-**L**anguages-**D**eveloper-**S**andbox